### PR TITLE
Topic page pagination tests (RTL and LTR)

### PR DESCRIPTION
**Pull request by LilyL0u**
Tests the pagination on topics pages.
Does not run the tests if there is only one page.
Tests cover, in a user journey starting from page 1:

-Should show pagination if there is more than one page in the pageCount in the data
-Should have the correct maximum page number
[the following checks that the correct url is navigated to but also that the page isn't blank]
-Page 2 button navigates to page 2
-Next button navigates to page 3 if there are more than 2 pages
-Last page number button navigates to the max page count numbered page
-Previous page button navigates to the previous page (2nd to last in this journey)
-Page 1 button navigates to back 1

-Above 400px width, 'Page x of y' is not visible
-Below 400px width, 'Page x of y' is visible

Restructured tests so the user journey tests are first, ensuring it begins from a topic page.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [x] I have assigned this PR to the Simorgh project
- [x] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)

**Testing:**
Passes on test and live.

### CODE REVIEW

File: cypress/integration/pages/topicPage/tests.js [hunk 1]
```diff
@@ -4,48 +4,27 @@ export default ({ service, pageType }) => {
...
-        const numberOfItems = body.data.summaries.length;
-        cy.log(numberOfItems);
-        // Checks number of items on page
-        cy.get('[data-testid="topic-promos"]')
-          .children()
-          .its('length')
-          .should('eq', numberOfItems);
+        pageCount = body.data.pageCount;
       });
+      cy.log(`topic id${topicId}`);
```
Comment (eagerterrier) [inline cypress/integration/pages/topicPage/tests.js]: Sorry for being pedantic but shouldn't there be a space between id and $ to make this readable/copyable?

File: cypress/integration/pages/topicPage/tests.js [hunk 2]
```diff
@@ -4,48 +4,27 @@ export default ({ service, pageType }) => {
...
-      cy.log(currentpath);
+    let pageCount;
 
-      [topicId] = currentpath.split('topics/').pop().split('?');
-
-      cy.log(`topic id${topicId}`);
-    });
-
-    it('should render a H1, which contains/displays topic title', () => {
+    const ltrServices = ['persian', 'pashto', 'arabic', 'urdu'];
```
Comment (eagerterrier) [inline cypress/integration/pages/topicPage/tests.js]: Aren't these rtlServices?

**Review by eagerterrier — COMMENTED** [review]
Some trifling Qs

File: cypress/integration/pages/topicPage/tests.js (hunk 1 above)
Comment (LilyL0u) [reply cypress/integration/pages/topicPage/tests.js]: added space

File: cypress/integration/pages/topicPage/tests.js (hunk 2 above)
Comment (LilyL0u) [reply cypress/integration/pages/topicPage/tests.js]: Yes oops

Comment (LilyL0u) [reply cypress/integration/pages/topicPage/tests.js]: Changed

**Review by RetiredBlob — APPROVED** [review]

**Review by rebeccamcginn — APPROVED** [review]

**Pull request merged**